src/DRAMPower/DRAMPower/memspec/MemSpecDDR5.cpp

```diff
@@ -50,6 +51,9 @@ MemSpecDDR5::MemSpecDDR5(nlohmann::json &memspec)
     memPowerSpec[VPP].iXX2P     = (parseUdoubleWithDefault(memspec["mempowerspec"]["ipp2p"], "ipp2p"));
     memPowerSpec[VPP].iXX3P     = (parseUdoubleWithDefault(memspec["mempowerspec"]["ipp3p"], "ipp3p"));
 
+    // TODO: have different spec for vddq?
+    memPowerSpec.push_back(MemPowerSpec());
+    memPowerSpec[VDDQ] = memPowerSpec[VDD];
 
```

Comment:
@lsteiner-tukl what you think about this? We aren't parsing the VDDQ power domain, but we use it in the IO power calculation class. Right now I just made it the same as VDD but i'm not sure what's the correct approach

Reply:
We need to add it to the mempowerspec, the standard specified currents for all three voltage domains.